src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,117 @@
 package core.basesyntax;
 
+import java.util.Arrays;
```

Comment:
Don't use Objects, Arrays, or any other util class. Since you're implementing your own ArrayList, you should avoid using Java's built-in Arrays utility class methods like `Arrays.toString()` in your `toString` method.